app/views/comments/destroy.js.erb

```diff
@@ -0,0 +1,4 @@
+$("#<%= dom_id(@comment) %>").remove()
```

Comment:
should we have a `can? :destroy` line here?

Reply:
I don't think it is worth it.
If we are rendering this view it is bc we went through `CommentsController#destroy`, so we have that permission.

Now that you mention it, I am not sure if it is worth it to keep https://github.com/dradis/dradis-ce/pull/400/files#diff-25ea4ccfaab28d70bcffa9056829dd06R1.
I will keep it bc it does not hurt, but we could remove it in my opinion.

Reply:
Good point